src/amp-events.js

```diff
@@ -15,17 +15,20 @@
  */
 
 /**
-* Enum used to specify custom Amp Events
-*
-* @enum {string}
-*/
+ * Common AMP events.
+ * @enum {number}
+ */
 export const AmpEvents = {
-  VISIBILITY_CHANGE: 'amp:visibilitychange',
-  DOM_UPDATE: 'amp:dom-update',
-  BUILT: 'amp:built',
-  ATTACHED: 'amp:attached',
-  STUBBED: 'amp:stubbed',
-  LOAD_START: 'amp:load:start',
-  LOAD_END: 'amp:load:end',
-  ERROR: 'amp:error',
+  // Values start at 801 for easier code searchability and to avoid conflation
+  // with HTTP status codes.
+  DOM_UPDATE: 801,
```

Comment:
Why are we changing to numbers?

Reply:
Travis build failed due to bundle-size. This saves 0.02KB. 😄 We'll end up doing the same with error messages which should save a bunch more.

Had to change back to strings to avoid conversion for event type.

Reply:
Actually realized `FixedLayer` doesn't need to dispatch DOM_UPDATE (at least for amp-bind) since it doesn't add new elements but just moves them on/off the transfer layer. 

Still would be nice to keep this change for a small size reduction.